src/cargo/ops/cargo_generate_lockfile.rs

```diff
@@ -170,7 +215,7 @@ pub fn update_lockfile(ws: &Workspace<'_>, opts: &UpdateOptions<'_>) -> CargoRes
                     &added[0].source_id().precise_git_fragment().unwrap()[..8],
                 )
             } else {
-                format!("{} -> v{}", removed[0], added[0].version())
+                format!("{} -> v{}{latest}", removed[0], added[0].version())
```

Comment:
Note that we calculate `latest` once, so if you have depend on different versions of a package, we won't show anything if your highest version is the latest.

Thoughts?

Reply:
I mean, the unfortunate version of this is if your direct dependencies require an older version whereas there is some indirect edge to the newest version, that is, it's not showing something that would be actionable.